src/swarm/util/RecordStream.d

```diff
@@ -159,6 +159,16 @@ public struct Record
         splitRecord(cast(ubyte[])buf, this.key, this.value);
     }
 
+    /// Ditto
+    private void deserialize ( Console.Input stream )
+    {
+        cstring data;
+        stream.readln(data);
```

Comment:
Would be nice to describe what the fix is exactly, in the commit message. (You mention the effect it has, but not what the fix actually is.)